plugins/vocabbuilder.koplugin/main.lua

```diff
@@ -1443,12 +1445,24 @@ local VocabBuilder = WidgetContainer:extend{
 
 function VocabBuilder:init()
     self.ui.menu:registerToMainMenu(self)
+    self:onDispatcherRegisterActions()
 end
 
 function VocabBuilder:addToMainMenu(menu_items)
     menu_items.vocabbuilder = {
         text = _("Vocabulary builder"),
         callback = function()
+            self:onShowVocabBuilder()
+        end
+    }
+end
+
+function VocabBuilder:prepareWidgetToShow()
```

Comment:
May be just :setupWidget() ? (I think we have other stuff doing it the same way).

Reply:
Done.